core/js/src/span_types.ts

```diff
@@ -6,3 +6,6 @@ export enum SpanTypeAttribute {
   TASK = "task",
   TOOL = "tool",
 }
+
+// Make SpanType any of the values of SpanTypeAttribute, eg llm, score, function, eval, task, tool
```

Comment:
nit: the docstring feels a bit unnecessary, but fine either way.

Reply:
That was a conversation I was having with Copilot lol. Removed